internal/config/client/client.go

```diff
@@ -26,6 +26,7 @@ type ClientConfig struct {
 }
 
 func BindAllEnv(v *viper.Viper) {
+	v.SetDefault("tenantId", "f790792c-cd0f-4304-91f3-3cb91feb019c")
```

Comment:
I wanted to re-use `DefaultTenantID` here but wasn't sure where if it's worth to do

Reply:
So I'm using the `defaults` package elsewhere for this behavior, so you can set this declaratively in the ConfigFile struct: 

```go 
type ClientConfigFile struct {
	TenantId string `mapstructure:"tenantId" json:"tenantId,omitempty" default:"f790792c-cd0f-4304-91f3-3cb91feb019c"`

	TLS ClientTLSConfigFile `mapstructure:"tls" json:"tls,omitempty"`
}
```

Reply:
I do think setting the client with a default tenant ID probably isn't great practice, since this will lead to a lot of confusing errors. I was thinking about changing the client implementation so that the client can use the tenant slug for connections instead of the UUID, which should address a lot of the issues here.

Reply:
> probably isn't great practice, since this will lead to a lot of confusing errors

Like what?